src/vuln_analysis/functions/cve_agent.py

```diff
@@ -36,9 +36,6 @@
 
 logger = LoggingFactory.get_agent_logger(__name__)
 
-ctx_state = contextvars.ContextVar("ctx_state", default="default_value")
-
-
```

Comment:
@IlonaShishov After this change, The following will break:
https://github.com/RHEcosystemAppEng/vulnerability-analysis/blob/ee9a733de5ebcd6bcafe8fe18940051e81a4c13d/src/vuln_analysis/tools/transitive_code_search.py#L16

Reply:
Thanks, updated the import to import context state from runtime_context.